content/ui-patterns/empty-states.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Empty states
+---
+
+import {Flex, Box, Text, BorderBox} from '@primer/components'
+
+Empty states are used to fill spaces when no content has been added yet, or is temporarily empty due to the nature of the feature. These guidelines demonstrate best practices for using the blankslate component and designing empty states. If you're looking for guidelines on implementation, please refer to [Primer CSS](https://primer.style/css/components/blankslate).
+
+## The blankslate component
+The blankslate is made up of several elements that work together to inform the user about a feature and how to proceed forward. Below are the different elements of the component and how to modify them.
+
+![blankslate anatomy](https://user-images.githubusercontent.com/6846673/62806713-78795380-baa8-11e9-9cbd-1a56fef3247b.png)
+
+### Graphic
+The graphic can bring delight, preview an interface element, or represent the goal of the feature. Graphics should be placed intentionally and with thought about the intention of the content. Graphics also differ in meaning and appeal to the user, which is why the blankslate component has multiple variations. These variations are outlined later on this page.
+
+### Primary Text
+Use primary text to explain the purpose of the empty state, help users feel comfortable to engage with the content, or begin a feature flow. Primary text should sound welcoming, human, and convey the intention of the feature.
+
+### Secondary text
+This optional text is used to inform the user about the feature in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should  be able to understand the general purpose of the feature and how it may help them accomplish a goal.
+
+### Primary action
+Blankslates can and are encouraged to use one primary action. This button should lead to a feature or component creation flow. Button copy should be keep brief and descriptive. If a button requires further specification, consider adding an Octicon.
+
+### Secondary action
+Secondary actions are optional and are represented by a text link located below the primary action button. A secondary action is used to direct a user to additional content about the feature. This might look like "[Learn more about X](#)" or "[Check out the guide on X](#)" or simply "[Learn more](#)".
+
+### Border
+The border is invisible by default, but can be added to help define the structure of the blankslate component when needed. This can be particularly helpful in page layouts where the blankslate is not the only content on the screen.
```

Comment:
The only part that feels like it's missing here is a link to the implementation guidelines. As these are design guidelines it's fine that we're not showing _how_ to add a border, but it would be useful to link to the part of the (future) implementation docs that show the border variant from here.

Reply:
@ZainKho that link is https://primer.style/css/components/blankslate#add-border

Reply:
Will add!